retroshare-gui/src/gui/qss/stylesheet/qss.default

```diff
@@ -138,6 +138,7 @@ ForumsDialog, GxsForumThreadWidget
 	qproperty-textColorUnreadChildren: darkgray;
 	qproperty-textColorNotSubscribed: black;
 	qproperty-textColorMissing: darkRed;
+	qproperty-textColorPinned: darkOrange;

```

Comment:
Of course somebody has to change it to what ordinary instance could use as usable. I thought background is plain white and darkorange was not an unwise choice.
![my_colors](https://user-images.githubusercontent.com/14850730/90984192-75b3a880-e5ae-11ea-841d-47025bfb9150.png)
![default_qss_colors](https://user-images.githubusercontent.com/14850730/90984201-8106d400-e5ae-11ea-921e-7c7e8dd3df10.png)